.github/workflows/test_mlagility.yml

```diff
@@ -58,6 +58,8 @@ jobs:
           benchit examples/cli/scripts/two_models.py
```

Comment:
Can we show a nice error message when the user tries to use flags that are incompatible with .onnx files? Example:

`benchit sample.onnx --analyze-only`

Reply:
Consider the scenario where someone runs:

```
benchit models/*/*.*
```

And we have a folder like `models/onnx_zoo` that is full of `.onnx` files.

If we throw errors on arguments that only work for .py but do nothing (including no harm) for .onnx then we will eliminate some use scenarios. For example if someone wanted to set `--script-args` for the `.py` files. 

Can we wait to see if anyone has a major problem or confusion with `--analyze-only` or other flags, before making this change?

Reply:
Filed https://github.com/groq/mlagility/issues/232